docs/development.md

```diff
@@ -146,14 +143,44 @@ This operand deploys the `template-validator` found in
 See the old `template-validator` [repository](https://github.com/kubevirt/kubevirt-template-validator)
 for more docs.
 
-#### `vm-console-proxy` operand
+#### `vm-console-proxy`
+
+Installs the VM console proxy found in `data/vm-console-proxy-bundle`.
+A pull request (PR) for updating the bundle is automatically created when a new version
+of the [vm-console-proxy](https://github.com/kubevirt/vm-console-proxy) is released.
 
-Installs the VM console proxy found in `data/vm-console-proxy-bundle`. A PR for updating the bundle is automatically created when a new version of the [vm-console-proxy](https://github.com/kubevirt/vm-console-proxy) is released.
+#### `tekton-pipelines`
 
-#### `tekton-pipelines` operand
+Installs the Tekton Pipelines found in `data/tekton-pipelines`.
+A pull request (PR) for updating the bundle is automatically created when a new version
```

Comment:
This is not accurate, pipelines will never get an update when new release of tekton tasks is released. All changes are done manually

Reply:
Good that you saw it, because this is what right now is documented.